src/DatePicker.php

```diff
@@ -107,12 +107,16 @@ public function registerClientScript()
 
         $options = !empty($this->clientOptions) ? Json::encode($this->clientOptions) : '';
 
+        $js[] = "$selector.datepicker($options);";
+
         if ($this->inline) {
+            if (!empty($this->clientEvents['changeDate'])) {
```

Comment:
Shouldn't be?: 

```php 
if(empty($this->clientEvents['changeDate'])) {
    $this->clientEvents['changeDate'] = "function (e){ jQuery('#$id').val(e.format());}";
}
```

Reply:
else //not empty
<input /> 
will not assigned value attr.
we lost your default handler

Reply:
if we set config  'inline' => true && changeDate = ...
js looks like:

```
;jQuery('#hidden_to').parent().datepicker({"autoclose":true,"format":"yyyy-mm-dd","startDate":"+1d"});
;jQuery('#hidden_to').parent().on('changeDate', function() {$("button", $(this).closest(".modal-dialog")).prop("disabled", false)});
;jQuery('#hidden_to').parent().on('changeDate', function (e){ jQuery('#hidden_to').val(e.format());});
```